docker-orchestration/sample-platform-python/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Installer for the CCExtractor sample platform Docker
+#
+# More information can be found on:
+# https://github.com/CCExtractor/sample-platform
+#
+dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+root_dir=$(cd "${dir}"/../ && pwd)
+clear
+date=$(date +%Y-%m-%d-%H-%M)
+
+echo "Welcome to the CCExtractor platform installer for docker!"
+echo ""
+echo "-------------------------------"
+echo "|   Installing dependencies   |"
+echo "-------------------------------"
+echo ""
+echo "* Updating package list"
+apt-get update
+echo "* Installing nginx, python, pip, kvm, libvirt and virt-manager"
+apt-get -q -y install nginx python python-dev python3-libvirt libxslt1-dev libxml2-dev python-pip qemu-kvm libvirt-dev virt-manager mediainfo
```

Comment:
It seems wiser to me to move this to the Dockerfile (I think most of this installer can actually be moved into a dockerfile when building...)

Reply:
fixed.